package.json

```diff
@@ -22,13 +22,14 @@
     },
     "main": "build/index.js",
     "scripts": {
-        "start": "yarn run serve",
+        "start": " yarn swagger-autogen api.astar.network && yarn run serve",
         "serve": "node build/index.js",
-        "dev": "ts-node-dev -r dotenv/config src/index.ts",
-        "build": "tsc --project tsconfig.json",
+        "dev": "yarn swagger-autogen && ts-node-dev -r dotenv/config src/index.ts",
+        "build": "yarn swagger-autogen && tsc --project tsconfig.json",
```

Comment:
Can you add a GitHub action that deploys the built Open API document to GitHub pages?